danielanatolie/drawerd
# mermaid input ready

Mermaid input is now ready. It is stored in the `mermaidInput` field in Tokenizer

**To test:**
Comment out line 44, line 48-51 in the Program class
Hit F5 to start running
The console will show the input for mermaid
Use it as input to the [live editor](https://mermaidjs.github.io/mermaid-live-editor/#) and you should get something like this:
![image](https://user-images.githubusercontent.com/20333350/66277411-f6609d80-e852-11e9-85b2-5c045b963f76.png)

`mermaidInput` is an array, to convert it to string, use `convertMermaidInputToString` in the Tokenizer class.
